test/test_test.c

```diff
@@ -480,6 +480,38 @@ static int test_single_eval(void)
            && TEST_mem_eq(p--, sizeof("456"), "456", sizeof("456"));
 }
 
+static int test_output(void)
+{
+    const char s[] = "1234567890123456789012345678901234567890123456789012"
+                     "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+    test_output_string("test", s, sizeof(s) - 1);
+    test_output_memory("test", (const unsigned char *)s, sizeof(s));
+    return 1;
+}
+
+static const char *bn_output_tests[] = {
+    NULL,
+    "0",
+    "-12345678",
+    "1234567890123456789012345678901234567890123456789012"
+        "1234567890123456789012345678901234567890123456789013"
+        "987657"
```

Comment:
Ew! So point is that you don't trust reader to be able to follow commas, right? And intention ought to be to "startle" reader with jagged formatting. Why not with parenthesis? E.g.
```
    ("1234567890123456789012345678901234567890123456789012"
     "1234567890123456789012345678901234567890123456789013"
     "987657")
```